### DISCUSSION TITLE
Alias Whitelist

### DISCUSSION THREAD

**Discussion by lv6vyzts93foqih3** [simple-login/app#526] [Feature Requests]
As a user, I want the option to enable a whitelist for each alias, so that I can restrict the use of the alias to sender addresses that I trust.

**Comment by nguyenkims** [comment]
@lv6vyzts93foqih3  Do you think this one is similar to https://github.com/simple-login/app/discussions/406?

**Reply by lv6vyzts93foqih3** [reply]
@nguyenkims Yes, I think this could be the same feature and users could chose to make it either a whitelist or a blacklist. The blacklist would probably make for a better default option because that way initially emails from all sender addresses would go through same as now.

**Comment by gogogohub** [comment]
This would be very useful for me. I like that I can block individual contacts but if I could whitelist a particular domain then it will matter less if my email address is shared out. I use one service that seems to have shared out my address a few times now and I don't appreciate the junk. I could do a new email address but I think they will share it out again.
